docs/source/algorithms/Fit-v1.rst

```diff
@@ -688,6 +693,118 @@ Output:
     Workspace2 ExpDecay.Height: 8.44
     Workspace2 ExpDecay.Lifetime: 1.40
 
+Comparison with scipy.optimize.curve_fit
+----------------------------------------
+
+The `scipy.optimize.curve_fit` function is a commonly used optimiser for fitting models to data.
+By default the `curve_fit` scaled the covariance matrix from the fit by the reduced chi-squared -
+this is equivalent to scaling the errorbars on the data points such that the reduced chi-squared is unity
+(the expectation value for a "good" fit). This can be useful if a fit is performed with unit weights
+or the weights are only relative. However for absolute weights (such as those calculated from the error on the counts
+using Poisson statistics) this is not valid.
+
+This leads to a discrepancy in the uncertainty on the best fit parameters and the confidence bounds
```

Comment:
I accept this sentence is quite clunky so I will have another go, however I might use slightly different wording as explained below - hope that's OK!
1. Use of the word `reported` here to me suggests something printed to the log etc. - perhaps `calculated` is a better word?
`fit parameters and the confidence bounds are reported`
2. I think we need to be very clear to distinguish `weights` used in the fit and `uncertainty/error `on the output of the fit here
 `Mantid uses the absolute errors for the fit parameters and for the best fit line (since v6.3)`
The weights used in the fit are the same for both `curve_fit` and `mantid` - it's just the difference lies in the interpretation of what those weights represent when calculating the error on the best fit parameters etc. To this end I will also clarify what I have written on Line 703!

Reply:
sounds good to me